File: 34-ingestion/python-sdk/ingestion.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def ingest_data(source, data):
    res = requests.post(f"{API}/ingestion",
        json={"source": source, "data": data},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Data ingested: {res.json()['data']['count']}")
    return res.json()["data"]

File: 33-ingestion/npm-package/ingestion.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function ingestData(source: string, data: any[]) {
  const res = await axios.post(`${API}/ingestion`, {source, data},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Data ingested:', res.data.data.count);
  return res.data.data;
}
